tests/h/storage_test.py

```diff
@@ -210,6 +210,20 @@ def test_it_allows_when_unscoped_group(
 
         assert result == models.Annotation.return_value
 
+    def test_it_allows_outside_scope_when_enforce_scope_is_False(
+        self, pyramid_request, pyramid_config, group_service, scoped_open_group, models
+    ):
+        scoped_open_group.enforce_scope = False
+        group_service.find.return_value = scoped_open_group
+
```

Comment:
```suggestion
```

Split the test into just three sections (arrange, act, assert), makes it clearer what's going on.